titiler/application/setup.py

```diff
@@ -0,0 +1,45 @@
+"""Setup titiler-application."""
+
+from setuptools import find_namespace_packages, setup
+
+with open("README.md") as f:
+    long_description = f.read()
+
+inst_reqs = [
+    "titiler-core",
+    "brotli-asgi>=1.0.0",
+    "python-dotenv",
+]
+extra_reqs = {
+    "test": ["pytest", "pytest-cov", "pytest-asyncio", "requests"],
+    "server": ["uvicorn[standard]>=0.12.0,<0.14.0"],
```

Comment:
Uvicorn is optional (e.g when running the application in AWS Lambda you don't need Uvicorn)